### ISSUE TITLE
question regarding to the segmentation map

### ISSUE THREAD

**Issue by xinario** [GalAvineri/ISIC-Archive-Downloader#12]
First of all, thanks for sharing this great repo. It makes downloading the dataset way much  easier.

When I tried to download the segmentation map using the code, it will always get stuck at 51%. The number of downloaded segmentation map is always 13779. I'm wondering if that's because the annotation is not complete or I'm missing something.

**Comment by GalAvineri** [comment]
Hello!
I'm very glad that this code is helping you :)

The issue you describe is caused by a bug in my code,
which is in turn caused by the fact no all images have segmentation.

Basically, when the script runs into a few images that do not have segmentations, it gets stuck.

I have an idea regarding how to solve the bug :) it's quite simple really.
Sadly I just haven't found time to implement it yet

**Comment by GalAvineri** [comment]
I'm happy to say that I found some spare time to work on this.
The bug i assumed existing in my code was indeed there, and i fixed it.
I also encountered another problem with the isic archive - some images have segmentations, but they are not downloadable.
This means that when you request them with the web api they provide, it won't be returned.
So i just skip those images, and the whole download now continues.

##So in conclusion:
Some images either have no available segmentations.
I now skip those and the download continues smoothly.
At the end of the download procedure i alert how many segmentation images have not been able to be fetched.

Can you try it and let me know if it works for you as well? :)

**Comment by xinario** [comment]
Thanks, it works very well.

**Comment by GalAvineri** [comment]
I'm glad to hear! :)
